noether/src/segmentation_server.cpp

```diff
@@ -55,28 +57,36 @@ class SegmentationAction
     bool laplacian_edge_smoothing = cfg.laplacian_edge_smoothing;
     bool laplacian_boundary_smoothing = cfg.laplacian_boundary_smoothing;
 
-
     // Set defaults if invalid data given (or not set). Note: No guarantee of fitness of defaults
     curvature_threshold = (curvature_threshold < 0.0001) ? 0.05 : curvature_threshold;
-    min_cluster_size = (min_cluster_size ==0) ? 500 : min_cluster_size;
+    min_cluster_size = (min_cluster_size == 0) ? 500 : min_cluster_size;
     max_cluster_size = (max_cluster_size == 0) ? 1000000 : max_cluster_size;
+    neighborhood_size = (neighborhood_size < 0.0001) ? 0.05 : neighborhood_size;
```

Comment:
The word "size" suggest to me that this is an integer

Reply:
I can I can change it to neighborhood_radius.